scripts/mkcloud

```diff
@@ -334,6 +334,8 @@ function sshrun
         export ipmi_ip_addrs=$ipmi_ip_addrs;
         export bmc_user=$bmc_user;
         export bmc_password=$bmc_password;
+        export want_ses_version=$want_ses_version;
+        export want_external_ceph=$want_external_ceph;
```

Comment:
All `$want_`* variables are passed to `$mkcconf` in line 340. So `$want_`* variables do not need to be listed here explicitly. Just trust the magic :)